### PR TITLE
tests: bluetooth: tester: Fix not marking chan as unused on disconnect

### PR DESCRIPTION
**Pull request by sjanc**
This fix not being able to re-connect channel after disconnect.

Signed-off-by: Szymon Janc <szymon.janc@codecoup.pl>

### CODE REVIEW

**Comment by jhedberg** [comment]
This needs a corresponding issue in order to be accepted due backporting.

**Review by jhedberg — APPROVED** [review]

**Comment by sjanc** [comment]
Yes, I've created issue for qualification bugs in 2.7
https://github.com/zephyrproject-rtos/zephyr/issues/39882

**Review by alwa-nordic — APPROVED** [review]

**Comment by jhedberg** [comment]
> Yes, I've created issue for qualification bugs in 2.7

I'm not sure how well that works with the "Fixes ..." encoding that's used in PRs to automatically close the corresponding issue upon merging. Having a single issue for multiple PRs doesn't seem to be supported by this mechanism.

**Comment by jhedberg** [comment]
This needs one more review from a qualifying reviewer before GitHub lets us merge it

**Comment by sjanc** [comment]
> > Yes, I've created issue for qualification bugs in 2.7
> 
> I'm not sure how well that works with the "Fixes ..." encoding that's used in PR descriptions to automatically close the corresponding issue upon merging. Having a single issue for multiple PRs doesn't seem to be supported by this mechanism.

I guess we can skip auto-backporting and later on I can make single PR with all required fixed..

**Review by Thalley — APPROVED** [review]

**Pull request merged**
